src/Sparrow.Server/Platform/Win32/Win32MemoryProtectMethods.cs

```diff
@@ -24,8 +25,12 @@ public static unsafe class Win32MemoryProtectMethods
             UIntPtr dwLength
         );
 
-        [DllImport("kernel32.dll", SetLastError = true)]
-        public static extern bool DiscardVirtualMemory(void* lpAddress, UIntPtr size);
+        //[DllImport("kernel32.dll", SetLastError = true, CallingConvention = CallingConvention.Winapi)]
+        //[SecurityCritical]
+        //public static extern bool PrefetchVirtualMemory(IntPtr hProcess, UIntPtr NumberOfEntries, MemorySegment* VirtualAddresses, ulong Flags);
+
+        //[DllImport("kernel32.dll", SetLastError = true)]
+        //public static extern bool DiscardVirtualMemory(void* lpAddress, UIntPtr size);
```

Comment:
Better to delete, rather than comment the code

Reply:
Yeah. Was merging and wanted to be sure it was not needed and forgot to delete afterwards